[content/en] Add contributor experience survey announcement and blog post (#6120)
open-telemetry/opentelemetry.io: content/en/blog/2025/contribex-survey.md

---
title: The OpenTelemetry Contributor Experience Survey is open!
linkTitle: ContribEx Survey is open
date: 2025-02-17
author: >-
  [Pablo Baeyens](https://github.com/mx-psi) (Datadog)
sig: Contributor Experience
cSpell:ignore: Baeyens Outreachy
---

Since its founding, OpenTelemetry has had [thousands of contributors][3] from

> all around the world, including end users, vendors and students. [Last

Reviewer: ```suggestion
around the world, including end users, vendors, and students. [Last
```

summer][1] some members of the OpenTelemetry project came together to create

> [Contributor Experience SIG][2], a SIG focused on making sure contributors of
> all types can have a great experience with the OpenTelemetry Project. Since
> then, the SIG has been working on a number of initiatives, including an
> [Outreachy project][5] to [create up to date contributor documentation][4].

Reviewer: ```suggestion
the [Contributor Experience SIG][2], which is focused on ensuring  
that contributors of all types have a great experience with the OpenTelemetry  
project.  

Since then, the SIG has worked on several initiatives, including an  
[Outreachy project][5] to develop [up-to-date contributor docs][4].
```

---
title: The OpenTelemetry Contributor Experience Survey is open!
linkTitle: ContribEx Survey is open
date: 2025-02-17
author: >-
  [Pablo Baeyens](https://github.com/mx-psi) (Datadog)
sig: Contributor Experience

> # prettier-ignore

Reviewer: ```suggestion
```

cSpell:ignore: Baeyens Outreachy
---

Since its founding, OpenTelemetry has had [thousands of contributors][3] from
all around the world, including end users, vendors and students. [Last
summer][1] some members of the OpenTelemetry project came together to create

> [Contributor Experience SIG][2]; a SIG focused on improving the experience of
> contributors to the OpenTelemetry Project, to make sure contributors of all
> types can have a great experience. Since then, the SIG has been working on a

Reviewer 2: It was a little repetitive. 

```suggestion
[Contributor Experience SIG][2], a SIG focused on making sure contributors of all
types can have a great experience with the OpenTelemetry Project. Since then, the SIG has been working on a
```

project][5] to develop [up-to-date contributor docs][4].

To inform future initiatives and help us understand the current state of the

> contributor experience, we are launching today our first official Contributor

Reviewer 3: ```suggestion
contributor experience, today we are launching our first official Contributor
```

contributor experience, we are launching today our first official Contributor
Experience Survey. If you have ever contributed to the project, be it with code,

> documentation or in a different way, we want to hear from you! The survey should

Reviewer 2: ```suggestion
documentation, or in a different way, we want to hear from you! The survey should
```

> should take between five and ten minutes, and is **open from February 17th up
> until March 17th**.

Reviewer: ```suggestion
should take between five and ten minutes, and is **open from February 17th to
March 17th**.
```

March 17th**.

The survey is anonymous; only [maintainers of the Contributor Experience SIG][7]
will see your full individual answers. We will share an aggregate analysis of
the survey results once completed, with a special focus to avoid data
re-identification.

> You can **access the survey [here][6]**. If you have any questions or concerns,
> please reach out to Pablo Baeyens on the CNCF Slack or via the email listed on
> the first page of the survey.

Reviewer 2: Rewording to get around using "here" for link text.

```suggestion
We invite you to **access the [survey][6]** and share your experience. If you have any questions or concerns,
please reach out to Pablo Baeyens on the [CNCF Slack][8] or using the email listed on
the first page of the survey.
```

[1]: https://github.com/open-telemetry/community/pull/2162
[2]: https://github.com/open-telemetry/sig-contributor-experience
[3]:
  https://opentelemetry.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1
[4]: https://github.com/open-telemetry/sig-contributor-experience/issues/16
[5]: https://www.outreachy.org/
[6]:
  https://docs.google.com/forms/d/e/1FAIpQLScoG279ZhRuMu8J_8BebGEVtMOS8BgD9cpQUJ6xSnNIAUtedw/viewform?usp=header
[7]:

>   https://github.com/orgs/open-telemetry/teams/sig-contributor-experience-maintainers

Reviewer 2: ```suggestion
  https://github.com/orgs/open-telemetry/teams/sig-contributor-experience-maintainers
  [8]: https://slack.cncf.io/
```

---
title: The OpenTelemetry Contributor Experience Survey is open!
linkTitle: ContribEx Survey is open
date: 2025-02-17
author: >-
  [Pablo Baeyens](https://github.com/mx-psi) (Datadog)
sig: Contributor Experience
cSpell:ignore: Baeyens Outreachy
---

> **TL;DR: Help us improve the contributor experience by taking our [survey][6].**

Reviewer: How about this?

```suggestion
{{% alert-md title="TL;DR" color="secondary" %}}

Take our [OpenTelemetry Contributor Survey][6] and help us **enhance the contributor experience**.
We appreciate your time and feedback!

{{% /alert-md %}}
```
